kubetest/kops_test.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import "testing"
+
+///
+// TODO:(@chrislovecnm) we should mock out the aws interface so we do not make a aws API call
```

Comment:
I think we should do one of (1) just remove the skipped tests, (2) implement the mock, (3) make the function testable by splitting it up.  I'm not sure there's much to test in option 3, TBH.